test/vmtests/vmtests/conftest.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+import random
+import shutil
+import string
+import tempfile
+from pathlib import Path
+from typing import Generator
+
+import docker
+import pytest
+from docker import DockerClient
+
+SCRIPT_PATH = Path(__file__).parent
+TEST_CONFIGS_DIR = SCRIPT_PATH.joinpath("../../../toolkit/tools/pkg/imagecustomizerlib/testdata")
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption("--keep-environment", action="store_true", help="Keep the resources created during the test")
+    parser.addoption("--core-efi-azl2", action="store", help="Path to Azure Linux 2.0 core-efi qcow2 image")
```

Comment:
why only azl2?

Reply:
Others will be added in the future. Just keeping it simple for now.